src/Screen/AirportsScreen/index.tsx

```diff
@@ -28,6 +33,16 @@ export const AirportsScreen: FC<AirportsScreenProps> = (props) => {
   const onScroll = Animated.event([{ nativeEvent: { contentOffset: { y } } }], {
     useNativeDriver: true,
   });
+  const emptyComponent = () => {
+    return (
+      <View style={Styles.emptyResult}>
+        <SVGNotFound color={theme.color.background} />
+        <Text style={Styles.textEmptyResult}>Empty</Text>
+      </View>
+    );
+  };
+
+  const contentContainerStyleFlatList = !airportsListData.length && { flex: 1 };
```

Comment:
please not use unline styles, use existing styles Styles.container instead of flex: 1